src/pages/index.js

```diff
@@ -21,38 +25,34 @@ import Section from "../components/Section.js";
 import { PopupWithForm } from "../components/PopupWithForm.js";
 import { PopupWithImage } from "../components/PopupWithImage.js";
 
-const cardsSection = new Section(
-  {
-    data: initialCards,
-    renderer: (item) => {
-      const cardElement = createCard(item);
-      cardsSection.addItem(cardElement);
-    },
-  },
-  cardList
-);
-cardsSection.renderItems();
+import { Api } from "../components/Api.js";
 //
 export const cardPopup = new PopupWithImage(modalCard);
 cardPopup.setEventListeners();
 
-const profileEddited = new UserInfo(profileTitle, profileSubtitle);
+const profileEdited = new UserInfo(profileTitle, profileSubtitle);
 
 const profilePopup = new PopupWithForm(profileEditModal, {
   handleFormSubmit: (obj) => {
-    profileEddited.setUserInfo(obj);
+    //added:
+    api.editProfile(obj);
+
+    profileEdited.setUserInfo(obj);
```

Comment:
You need to update `DOM` only in blocks `then` after a successful response from the server so that when there is an error it wouldn’t change


**Please, correct it everywhere**